tests/attr/test_integrated_gradients_basic.py

```diff
@@ -253,86 +264,101 @@ def _assert_batched_tensor_multi_input(
 
     def _compute_attribution_and_evaluate(
         self,
-        model,
-        inputs,
-        baselines=None,
-        target=None,
-        additional_forward_args=None,
-        type="vanilla",
-        approximation_method="gausslegendre",
-    ):
+        model: nn.Module,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Optional[Any] = None,
+        target: Optional[int] = None,
+        additional_forward_args: Optional[Any] = None,
+        type: Optional[str] = "vanilla",
+        approximation_method: str = "gausslegendre",
+    ) -> Tuple[Tensor, ...]:
         r"""
             attrib_type: 'vanilla', 'smoothgrad', 'smoothgrad_sq', 'vargrad'
         """
         ig = IntegratedGradients(model)
-        if not isinstance(inputs, tuple):
-            inputs = (inputs,)
-
-        if baselines is not None and not isinstance(baselines, tuple):
-            baselines = (baselines,)
+        inputs_tuple: Tuple[Tensor, ...]
```

Comment:
If we avoid the explicit type hints here, can we leave the original code? Or is there an error that requires these hints?

Reply:
Casting solved these annoyances, the changes are tighter now.